src/main/java/io/strimzi/kafka/bridge/Application.java

```diff
@@ -23,28 +33,73 @@
+                    ar.result().getString("amqp.certDir", AmqpConfig.DEFAULT_CERT_DIR));
+
+            HttpConfig httpConfig = new HttpConfig(ar.result().getBoolean("http.enabled", HttpConfig.DEFAULT_HTTP_ENABLED),
+                    ar.result().getString("http.host", HttpConfig.DEFAULT_HOST),
+                    ar.result().getInteger("http.port", HttpConfig.DEFAULT_PORT));
+
+            KafkaConsumerConfig kafkaConsumerConfig = new KafkaConsumerConfig(ar.result().getString("kafka.consumer.autoOffsetReset", KafkaConsumerConfig.DEFAULT_AUTO_OFFSET_RESET));
+            KafkaProducerConfig kafkaProducerConfig = new KafkaProducerConfig(Integer.toString(ar.result().getInteger("kafka.producer.acks", Integer.parseInt(KafkaProducerConfig.DEFAULT_ACKS))));
+
+            KafkaConfig kafkaConfig = new KafkaConfig(ar.result().getString("kafka.bootstrapServers", KafkaConfig.DEFAULT_BOOTSTRAP_SERVERS),
+                    kafkaProducerConfig, kafkaConsumerConfig);
+
+            AmqpBridgeConfig amqpBridgeConfig = new AmqpBridgeConfig(kafkaConfig, amqpConfig);
+            if (amqpBridgeConfig.getEndpointConfig().isEnabled()) {
+                AmqpBridge amqpBridge = new AmqpBridge(amqpBridgeConfig);
+                vertx.deployVerticle(amqpBridge, done -> {
+                    if (done.succeeded()) {
+                        log.debug("AMQP verticle instance deployed [{}]", done.result());
+                    } else {
+                        log.debug("Failed to deploy AMQP verticle instance", done.cause());
+                    }
+                });
+            }
+
+            HttpBridgeConfig httpBridgeConfig = new HttpBridgeConfig(kafkaConfig, httpConfig);
+            if (httpBridgeConfig.getEndpointConfig().isEnabled()) {
+                HttpBridge httpBridge = new HttpBridge(httpBridgeConfig);
+                vertx.deployVerticle(httpBridge, done -> {
+                    if (done.succeeded()) {
+                        log.debug("HTTP verticle instance deployed [{}]", done.result());
+                    } else {
+                        log.debug("Failed to deploy HTTP verticle instance", done.cause());
+                    }
+                });
+            }
+        });
+    }
 
-        HttpBridgeConfig httpBridgeConfig = HttpBridgeConfig.fromMap(System.getenv());
-        if (httpBridgeConfig.getEndpointConfig().isEnabled()) {
-            HttpBridge httpBridge = new HttpBridge(httpBridgeConfig);
-            vertx.deployVerticle(httpBridge, done -> {
-                if (done.succeeded()) {
-                    log.debug("HTTP verticle instance deployed [{}]", done.result());
-                } else {
-                    log.debug("Failed to deploy HTTP verticle instance", done.cause());
-                }
-            });
+    private static String getFilePath(String arg) {
+        String[] confFileArg = arg.split("=");
```

Comment:
It would be good if we can handle both `--config-file=xxx` and `--config-file xxx`